README.md

```diff
@@ -15,6 +15,9 @@ GOV.UK Pay Frontend application (Node.js)
 | `WORLDPAY_3DS_FLEX_DDC_LIVE_URL`       |    X     |                                                               | URL for Device Data Collection (DDC) initiation in LIVE                                                                                               |
 | `WORLDPAY_3DS_FLEX_CHALLENGE_TEST_URL` |    X     | `https://secure-test.worldpay.com/shopper/3ds/challenge.html` | Pointing to Worldpay's TEST 3ds flex challenge URL.                                                                                                   |
 | `WORLDPAY_3DS_FLEX_CHALLENGE_LIVE_URL` |    X     |  `https://centinelapi.cardinalcommerce.com/V2/Cruise/StepUp`  | Pointing to Worldpay's LIVE 3ds flex challenge URL.                                                                                                   |
+| `CSP_USE_CARD_PAYMENT_RULES`           |          | false/undefined                                               | Apply card payment contest security policy headers.                                                                                                   |
```

Comment:
What exactly do we mean by `CARD_PAYMENT_RULES` here? Does it mean _just_ the enter card details page (in which case the name should be more specific) or all pages related to card payments (which is redundant because that’s all frontend does)?

Reply:
👍  have renamed `CSP_SEND_HEADER`